client/my-sites/comments/comment/utils.js

```diff
@@ -0,0 +1,74 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get, includes } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { decodeEntities } from 'lib/formatting';
+
+/**
+ * Return the comment author display name, or "Anonymous" if the comment author has no name.
+ *
+ * @param {Object} comment A comment object.
+ * @returns {String} The comment author display name.
+ */
+export const getAuthorDisplayName = comment =>
+	decodeEntities( get( comment, 'author.name', translate( 'Anonymous' ) ) );
```

Comment:
If we only have one usage I'd probably move this to the component that uses it. This can also work as a global selector.

Reply:
To be honest, I simply copy-pasted the utils file I used in the other refactor attempt, with the intention to use what I could, and eventually delete what I ended up not using.

This function in particular seems exactly one of those that I don't really need, compared to the previous design, as we now have only one Author block instead of two.